creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -531,11 +534,16 @@ class SnapbackSM {
         newReplicaSetSPIds.push(this.peerSetManager.endpointToSPIdMap[endpt])
       }
 
+      const startTimeMs = Date.now()
       await this.audiusLibs.contracts.UserReplicaSetManagerClient.updateReplicaSet(
         userId,
         newReplicaSetSPIds[0], // primary
         newReplicaSetSPIds.slice(1) // [secondary1, secondary2]
       )
+      const timeElapsedMs = Date.now() - startTimeMs
+      this.log(
```

Comment:
ah i didnt realize you meant the bottleneck is literaly in the chain call, great call to log